Aardvark/ARKLogType.h

```diff
@@ -0,0 +1,20 @@
+//
+//  ARKLogType.h
+//  Aardvark
+//
+//  Created by Evan Kimia on 10/22/15.
+//  Copyright © 2015 Square, Inc. All rights reserved.
```

Comment:
All rights are not reserved. This line should read: "Copyright 2015 Square, Inc."

Please also include the following lines (as all the other files in Aardvark have):
```
//
//  Licensed under the Apache License, Version 2.0 (the "License");
//  you may not use this file except in compliance with the License.
//  You may obtain a copy of the License at
//
//    http://www.apache.org/licenses/LICENSE-2.0
//
//  Unless required by applicable law or agreed to in writing, software
//  distributed under the License is distributed on an "AS IS" BASIS,
//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
//  See the License for the specific language governing permissions and
//  limitations under the License.
//
```